pylatex/section.py

```diff
@@ -16,25 +17,37 @@ class Section(Container):
     #: A section should normally start in its own paragraph
     end_paragraph = True
 
+    #: Default prefix to use with Marker
+    marker_prefix = "sec"
+
     #: Number the sections when the section element is compatible,
     #: by changing the `~.Section` class default all
     #: subclasses will also have the new default.
     numbering = True
 
-    def __init__(self, title, numbering=None, **kwargs):
+    def __init__(self, title, numbering=None, label=True, **kwargs):
         """
         Args
         ----
         title: str
             The section title.
         numbering: bool
             Add a number before the section title.
+        label: Label or bool
+            Can set a label manually or use a boolean to set
+            preference between automatic or no label
         """
 
         self.title = title
 
         if numbering is not None:
             self.numbering = numbering
+        if isinstance(label, Label):
```

Comment:
Can you also handle a `label` argument that's a string? i.e. automatically make a `Label` out of it.